x-pack/plugins/apm/common/constants.ts

```diff
@@ -41,3 +43,8 @@ export const ERROR_LOG_STACKTRACE = 'error.log.stacktrace';
 export const ERROR_EXC_MESSAGE = 'error.exception.message';
 export const ERROR_EXC_STACKTRACE = 'error.exception.stacktrace';
 export const ERROR_EXC_HANDLED = 'error.exception.handled';
+
+export const TRANSACTION_TYPE_MAP: StringMap<string> = {
+  request: 'Request',
+  'page-load': 'Page load'
+};
```

Comment:
This doesn't really fit in here, does it? All the others are constants for ES properties.